prac_05/emails.py

```diff
@@ -1,4 +1,21 @@
 """
 Time estimate: 15 minutes
-actual:
-"""
\ No newline at end of file
+actual: 35 minutes
+"""
+
+email_to_name = {}
+
+email_address = input("Email: ")
+while email_address != "":
+    parts = email_address.split("@")
```

Comment:
parts and name are used multiple times for different reasons.

The 4 (10 -> 13) can be on the same line:
name = " ".join(email_adress.split("@")[0].split(".")).title()
^ is more compact but harder to read.

Reply:
Thanks for that, was having a hard time making it more concise.